packages/angular_devkit/core/src/index.ts

```diff
@@ -6,6 +6,7 @@
  * found in the LICENSE file at https://angular.io/license
  */
 import * as experimental from './experimental';
+import * as jobs from './jobs/index';
```

Comment:
Can the jobs/scheduler API be in experimental? Alternatively it could be a separate package.

I feel it is almost inevitable that some of the API will change as we use it, and do not want to be tied to semver for it.

Reply:
Done.